Makefile

```diff
@@ -0,0 +1,43 @@
+DEFAULT_VERSION = 'latest'
```

Comment:
do we still needs this make file? isn't it covered with test-action already? or do you think it is needed for local development?

Reply:
Yep, this is needed for local development. Passing parameters is very tedious especially in bash programs as they have to properly ordered.